github/compliance/report.py

```diff
@@ -887,7 +887,7 @@ def close_issues(
                 "scanned element matching its digest."
             )
             issue.create_comment('closing, because scanned element no longer present in BoM')
-            issue.close()
+            github.compliance.issue.close_issue(issue)
```

Comment:
one more thing: I think the `close_issue` function would fit better into github.util

Reply:
otherwise. lgtm

Reply:
will mv 👍